deploy.yml

```diff
@@ -14,6 +14,9 @@ parameters:
 - name: gh_pages_subdirectory
   type: string
   default: ''
+- name: publish_docs
+  type: boolean
+  default: false
```

Comment:
By setting the default to `false`, you will prevent the `main` and `release` pipelines in `scipp` from publishing docs unless you also make changes there.
The easiest is probably to set it to `true` and then just have it as `false` in the next pipeline?

Reply:
I did for scippneutron, but I agree we want to avoid changing the others.